src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -739,11 +739,17 @@ private void Sort(object searchQuery = null)
         public List<FilterEntry> InitializeHostFilter()
         {
             var hostFilter = new List<FilterEntry>();
-            foreach (var host in PackageManagerClientViewModel.Model.GetKnownHosts())
+            try
             {
-                hostFilter.Add(new FilterEntry(host, Resources.PackageFilterByHost, Resources.PackageHostDependencyFilterContextItem, this));
+                foreach (var host in new List<string>())
```

Comment:
We are not using GetKnownHosts()? host is iterating an empty list here?

Reply:
Oh..I was testing what happens when the method returns without results, and that got in by mistake, corrected it. Thanks!